### PR TITLE
Updating global paramters procedure

### PR DESCRIPTION
**Pull request by KonradStaniec**
Proposal for procedure to update global parameters from this point and onwards.

### CODE REVIEW

File: bbn-test-4/parameters/README.md
Comment (vitsalis) [inline bbn-test-4/parameters/README.md:101]:
```suggestion
Given that the staking parameters are used by multiple entities running in a distributed
environement to validate staking and unbonding transactions,
all updates to the `global-params.json` must be made in well-defined and
transparent manner.
```

File: bbn-test-4/parameters/README.md
Comment (vitsalis) [inline bbn-test-4/parameters/README.md:112]:
```suggestion
To update parameters the following steps will be taken:
1. The Babylon team creates a PR in this repository with an updated `global-params.json` file.
The only allowed modification to this file is appending a new object to the `versions`
collection. The newly appended object must obey all rules defined in the previous paragraph.
2. All interested entities, for example, covenant signers, approve this PR. Each
approval is interpreted as being ready to validate transactions using the new `global-params.json`
introduced by the PR.
3. After enough approvals are gathered, the PR is merged.

Now the tip of the `main` branch contains the last version of staking parameters.
```

How many approvals should we require? Quorum?

**Review by vitsalis — APPROVED** [review]
Nice!

File: bbn-test-4/parameters/README.md
Comment (KonradStaniec) [reply bbn-test-4/parameters/README.md:112]: I would say that at minimum quorum of covenant signers must approve as this effectively say that we can can proceed with unbonding signing.

Though starting new params epoch with only quorum of covenants having updated increases our risk profile so maybe we can have time based condition. Given that parameters are not updated frequently we could go with something like:
- wait for 48 for approval
- if all approved before 48h - merge
- if after 48h quorum approved - merge and contact not responsive parties
- if after 48h there is no quorum - contact not responsive parties

**Pull request merged**
